pkg/config/tlscfg/flags.go

```diff
@@ -74,6 +75,8 @@ func (c ClientFlagsConfig) InitFromViper(v *viper.Viper) Options {
 	var p Options
 	if c.ShowEnabled {
 		p.Enabled = v.GetBool(c.Prefix + tlsEnabled)
+	} else {
+		log.Fatalf("ES_TLS_ENABLED is set to false unable to talk to server(s)")
```

Comment:
We try not to use std log package, as it makes the code not-testable. The preferred way is to return an error and make sure it is logged upstream where the logger is already available / initialized.

Reply:
but don't we want to crash instead of logging or we can log it from the logger and then crash it?

Reply:
the top-level logger (e.g. somewhere in main) can log fatal, but all other packages should be returning errors

Reply:
okay